3.3/.sti/bin/assemble

```diff
@@ -1,23 +1,66 @@
 #!/bin/bash
 
-# For SCL enablement
-source .bashrc
+# This script uses these environment variables:
+#
+# DISABLE_COLLECTSTATIC: if not empty, inhibits execution of 'manage.py collectstatic'.
 
-set -e
+function assemble() {
+  # For SCL enablement
+  source .bashrc
```

Comment:
@soltysh in bash, you should never write functions to show 'the flow' because everyone that will try to understand it will have to go to that functions anyway... if you get problem running gunicorn/django server... understanding the flow will not fix your problem...

we should explain the 'flow' in documentation, not inside bash scripts... people will not go read them to understand the flow but to fix problems